### PR TITLE
Feature/38/playground

### PR DESCRIPTION
**Pull request by MariluzHerrera**
This is all I could do regarding styling. I'm not familiar enough with bootstrap and CSS so this will need proper styling from someone more expert.

Sorry for the name of the branch and if it looks messy in terms of commits and changes on the files. I've been rebasing and resolving conflicts for an hour. It looks correct to me, but I think it's better if someone with more knowledge have a close look at this.

### CODE REVIEW

**Comment by MariluzHerrera** [comment]
[Users can see a styled dashboard view with all 10k ft data](https://trello.com/c/XQXcsNH0/48-users-can-see-a-styled-dashboard-view-with-all-10k-ft-data)

File: db/migrate/20200220164019_add_discipline_to_team_members.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddDisciplineToTeamMembers < ActiveRecord::Migration[6.0]
+  def change
+    add_column :team_members, :discipline, :string
+  end
+end
```
[the remark this answers is not in the document]
Comment (MariluzHerrera) [reply db/migrate/20200220164019_add_discipline_to_team_members.rb]: That's true. I saw you assigned the discipline on the role column. I'm happy to delete the discipline column then.

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -59,9 +55,4 @@ h2 {
   list-style: none;
   margin: 5px;
   padding: 0;
-=======
-.card {
-  background-color: #00567E;
-  box-shadow: 10px 10px 5px 0px rgba(0,0,0,0.30);
->>>>>>> f2ace1a... WIP:app/assets/stylesheets/application.css
```
[the remark this answers is not in the document]
Comment (MariluzHerrera) [reply app/assets/stylesheets/application.scss]: Yeah, sorry. I spent a good hour squashing commits and trying to clean the mess. I thought that was gone. I'll see if I can fix it....

File: db/migrate/20200224152354_remove_discipline_from_team_members.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class RemoveDisciplineFromTeamMembers < ActiveRecord::Migration[6.0]
+  def change
+    remove_column :team_members, :discipline
+  end
```
Comment (LornaHarwood) [inline db/migrate/20200224152354_remove_discipline_from_team_members.rb]: May I suggest doing a rollback instead of a remove_column? 
You can do this by the following:
- First you will need to add the `type` into this migration (:string - on line 3)
- The run a `rails db:rollback STEP=2` to rollback 2 migrations steps.
- This will take you to migration `add_metadata_fields_to_project`
- Delete the add and remove column migrations

Comment (MariluzHerrera) [reply db/migrate/20200224152354_remove_discipline_from_team_members.rb]: I need to do more reading on the db rollback as I'm not familiar enough. I'll do it in the afternoon.

Just to make sure I understood to avoid going back and forth with this again... You want me to delete the `role` column and add the `discipline` column again by doing the `db:rollback` (and then substitute all references to role for discipline). Also, for clarification purposes, are you sure you mean the `add_metadata_fields_to_project` migration file as the one I should change? I think the `role` column lives in `add_meta_fields_to_team_member` but as I said, I'm not familiar enough with rolling back the database, so I just want to double check. Thank you!

File: db/migrate/20200220164019_add_discipline_to_team_members.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+class AddDisciplineToTeamMembers < ActiveRecord::Migration[6.0]
```
Comment (erbridge) [inline db/migrate/20200220164019_add_discipline_to_team_members.rb]: Looks like you're still doing two unnecessary migrations in this commit. You should be able to drop both of them and have the same effect.

File: app/views/projects/index.html.erb [hunk 5]
```diff
@@ -1,11 +1,24 @@
...
+      <div class="card card-radius" >
+        <div class="card-body">
+          <h3><%= project.name %></h3>
+            <div class="members">
+            <% project.team_members.each do |team_member| %>
+              <ul class="member unstyled-list">
+                <% if team_member.thumbnail.present? %>
+                  <%= image_tag team_member.thumbnail, style:"width: 50px; border-radius: 50%; " %>
+                <% else %>
+                  <%= image_tag("anon.png", style:"width: 50px; border-radius: 50%; ")  %>
```
Comment (erbridge) [inline app/views/projects/index.html.erb]: Let's not style in the template. Use a class instead.

File: app/assets/stylesheets/application.scss [hunk 6]
```diff
@@ -30,7 +31,29 @@ h2 {
...
+  }
+
+.member {
+  margin: 10px;
+  font-size: 14px;
+}
+
+.unstyled-list {
+  list-style: none;
+  margin: 5px;
```
Comment (erbridge) [inline app/assets/stylesheets/application.scss]:
```suggestion
  margin: 0;
```

This class says that it's unstyled, so let's make it actually unstyled.

File: db/migrate/20200219152902_add_meta_fields_to_team_member.rb [hunk 7]
```diff
@@ -1,6 +1,6 @@
 class AddMetaFieldsToTeamMember < ActiveRecord::Migration[6.0]
   def change
-    add_column :team_members, :role, :string
+    add_column :team_members, :discipline, :string
```
Comment (erbridge) [inline db/migrate/20200219152902_add_meta_fields_to_team_member.rb]: You can't edit a migration that's already run on your live environment. Migration only run once, so your live version will have already run this migration. Instead, you need to create a new migration to do a rename of the column.

However, this PR is about styling the frontend. Renaming the database field probably belongs in a different commit.

File: db/migrate/20200220164019_add_discipline_to_team_members.rb (hunk 4 above)
Comment (MariluzHerrera) [reply db/migrate/20200220164019_add_discipline_to_team_members.rb]: I'm getting a bit confused with this. On my local I don't have those migration files (I rolled back the DB and deleted manually those 2 migration files), so I'm not sure why they're still there if I pushed my latest version to github.

File: app/views/projects/index.html.erb (hunk 5 above)
Comment (MariluzHerrera) [reply app/views/projects/index.html.erb]: I know we shouldn't use inline styling. I researched about how to do a class on an image_tag and I couldn't find an answer. That's why I finally did it in the template, knowing I (or someone who knows better front-end) would need to add a class.

File: db/migrate/20200219152902_add_meta_fields_to_team_member.rb (hunk 7 above)
Comment (MariluzHerrera) [reply db/migrate/20200219152902_add_meta_fields_to_team_member.rb]: OK. What I did was rolling the DB back to before I ran this file, edit the file and then run `rails db:migrate` again. That's what I understood from the previous suggestions on this PR. 

Shall I create a new PR with this commit then?

File: app/assets/stylesheets/application.scss (hunk 6 above)
Comment (MariluzHerrera) [reply app/assets/stylesheets/application.scss]: > This class says that it's unstyled, so let's make it actually unstyled.

I removed the margin and the dashboard looks worse (the letters on names and disciplines from different team members are touching each other, making very difficult to distinguish and read). Not sure what's the best option at this point. Any ideas?

File: db/migrate/20200220164019_add_discipline_to_team_members.rb (hunk 4 above)
Comment (erbridge) [reply db/migrate/20200220164019_add_discipline_to_team_members.rb]: To be clear, this was a comment on the first commit. You did delete the migrations, but it would have been better to have removed them from the commit that introduced them. This is now resolved, but I thought you might find an explanation helpful.

**Comment by erbridge** [comment]
@MariluzHerrera It looks like you had some trouble pulling the changes down after Lorna had updated the branch. I would expect to see a single commit for all of the work in this PR, not 6 involving some merge commits.
